agent/command/results_xunit_test.go

```diff
@@ -166,70 +165,72 @@ func TestXUnitParseAndUpload(t *testing.T) {
 		WorkDir:         filepath.Join(testutil.GetDirectoryOfFile(), "testdata", "xunit"),
 		NewExpansions:   &util.DynamicExpansions{},
 	}
-	conf.Task.TaskOutputInfo.TestResults.Version = 0
+	conf.Task.TaskOutputInfo.TestResults.Version = 2
```

Comment:
nit: could we consider naming and turning the versions into constants? Like 2 might be the only valid test results version, but putting that in a named constant makes it clearer that it's not just some random number.